docs/guide_slash_and_interactions.rst

```diff
@@ -0,0 +1,297 @@
+            await interaction.response.send_message(f"Your color is {str(color.name)}", ephemeral=True)
+
+Check out the full reference of decorators on Discord.py's documentation `here <https://discordpy.readthedocs.io/en/stable/interactions/api.html#decorators>`__.
+
+
+Groups & Subcommands
+--------------------
+Slash commands can also be grouped together into groups and subcommands.
+These can be used to create a more complex command structure.
+
+.. note::
+    Unlike text command groups, top level slash command groups **cannot** be invoked.
+
+.. code-block:: python
+
+    import discord
+    
+    from redbot.core import commands, app_commands
+
+    class MyCog(commands.Cog):
+        def __init__(self, bot):
+            self.bot = bot
+
+        group = app_commands.Group(name="zoo", description="Zoo related commands")
+
+        @group.command(name="add", description="Add an animal to the zoo")
+        @app_commands.describe(animal="The animal you want to add")
+        async def add_animal(self, interaction: discord.Interaction, animal: str):
+            await interaction.response.send_message(f"Added {animal} to the zoo", ephemeral=True)
+
+        @group.command(name="remove", description="Remove an animal from the zoo")
+        @app_commands.describe(animal="The animal you want to remove")
+        async def remove_animal(self, interaction: discord.Interaction, animal: str):
+            await interaction.response.send_message(f"Removed {animal} from the zoo", ephemeral=True)
+
+Arguments
+---------
+As shown in some of the above examples, we can amplify our slash commands with arguments.
+However with slash commands Discord allows us to do a few more things.
+Such as specifically select a channel that we'd like to use in our commands,
+we can do the same with roles and members.
+Let's take a look at how we can do that.
+
+.. code-block:: python
+
+    import discord
+
+    from redbot.core import commands, app_commands
+
+    class MyCog(commands.Cog):
+        def __init__(self, bot):
+            self.bot = bot
+
+        @app_commands.command()
+        @app_commands.describe(channel="The channel you want to use")
+        async def channel(self, interaction: discord.Interaction, channel: discord.TextChannel):
+            await interaction.response.send_message(f"Your channel is {channel.mention}", ephemeral=True)
+
+        @app_commands.command()
+        @app_commands.describe(role="The role you want to use")
+        async def role(self, interaction: discord.Interaction, role: discord.Role):
+            await interaction.response.send_message(f"Your role is {role.mention}", ephemeral=True)
+
+        @app_commands.command()
+        @app_commands.describe(member="The member you want to use")
+        async def member(self, interaction: discord.Interaction, member: discord.Member):
+            await interaction.response.send_message(f"Your member is {member.mention}", ephemeral=True)
```

Comment:
Calling the command the same thing as the argument makes it a little confusing that the `name="Desc"` in the describe deco applies to the parameter name, not the function name. The previous example doesn't have this issue, so I don't think it's a huge deal, but something to consider for this and following examples.